sites/helpers/google_tag_manager.py

```diff
@@ -0,0 +1,35 @@
+import pandas as pd
```

Comment:
⛏ I think it's fine to just name this file `gtm.py` instead. Generally, we tend to [avoid long module names & underscores](https://docs.python-guide.org/writing/structure/#modules). Since the function & dictionary defined in this file already have very clear names, overall readability won't be much affected. But happy to keep `google_tag_manager.py` if you feel strongly about it!

Reply:
👍